docs/FAQ.md

```diff
@@ -0,0 +1,58 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
```

Comment:
I'd change the question to something like:

> My branch did not move when I did `jj commit`, help! 

and the answer to:

> In `jj`, you need to explicitly move your branches using `jj branch set`.

I'd also add a separate question:

> I made a commit and `jj git push --all` says "Nothing changed" instead of pushing it. What do I do?

with an answer along the lines of

> `jj git push --all` pushes all *branches*, not all *revisions*. You have two options:
>
> - Using `jj git push --change` will automatically create a branch and push it.
>
> - Use `jj branch` commands to create or move a branch to either the commit you want to push or a descendant on it. Unlike `git`, `jj` doesn't do this automatically (see previous question).

Reply:
Done, but I used Martin's suggestion here https://github.com/martinvonz/jj/pull/1977#discussion_r1285070151